src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -39,13 +40,31 @@ function getRightOperationLabel(target: string): string {
 	return localize('nodePropertyViewRightOperation', 'Right operation: {0}', target);
 }
 
+function getTopPlanIsGreaterThanBottomPlanSummaryPointTemplate(rowName: string): string {
```

Comment:
These aren't going to be enough - for example it doesn't really make sense for QueryHash/QueryPlanHash values to be "greater than" the other. 

![image](https://user-images.githubusercontent.com/28519865/192585832-52d7750b-3e3c-4012-bf69-d12acb69178a.png)